rest-api/src/main/java/com/cloudbees/workflow/util/ModelUtil.java

```diff
@@ -56,6 +56,6 @@ public static String getFullItemUrl(String itemUrl) {
 
     public static String getRootUrl() {
         StaplerRequest currentRequest = Stapler.getCurrentRequest();
-        return currentRequest.getContextPath();
+        return (currentRequest != null) ? currentRequest.getContextPath() : "/";
```

Comment:
No, it's a placeholder here, to allow building a dummy URL string (for testing) when there is not a request context.